tcp-echo.go

```diff
@@ -172,7 +172,17 @@ func generateHTTPHandler(message string) func(w http.ResponseWriter, r *http.Req
 		log.Println(clientId+" - Received HTTP Raw Data:", body)
 		log.Printf(clientId+" - Received HTTP Data (converted to string): %s", body)
 
-		_, err = io.WriteString(w, message)
+		// Add request details to the response
+
+		httpMessage := message
+		httpMessage = httpMessage + "\nHTTP request details\n"
```

Comment:
```suggestion
		httpMessage += "\nHTTP request details\n"
```
and subsequent. Btw. you can use [string builder](https://pkg.go.dev/strings#Builder) for such use case, but it doesn't matter much in this case

Reply:
I followed the existing code style for this https://github.com/Kong/go-echo/blob/main/tcp-echo.go#L46

Let me know if you'd like it updating to `+=` or if we should follow the existing style

Reply:
I rather prefer not repeating myself, but tbh it doesn't matter much here